src/AuthenticationService.php

```diff
@@ -212,14 +212,13 @@ public function authenticate(ServerRequestInterface $request): ResultInterface
      * @param \Psr\Http\Message\ServerRequestInterface $request The request.
      * @param \Psr\Http\Message\ResponseInterface $response The response.
      * @return array Return an array containing the request and response objects.
-     * @psalm-return array{request: \Psr\Http\Message\ServerRequestInterface, response: \Psr\Http\Message\ResponseInterface}
+     * @return array{request: \Psr\Http\Message\ServerRequestInterface, response: \Psr\Http\Message\ResponseInterface}
```

Comment:
it might be better to put them on phpstan-return, as those are non trivial annotations IMO.

Reply:
Every major IDE and stan tools understand array shapes now.